src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -173,6 +173,22 @@ scans:
 
 #### Suffix
```

Comment:
Rename title to something like:

`### Supported modalities, quantitative maps and grouped scan collections`

BUT keep the term suffix - we're proposing that as a better term than "modality_label".

BUT NOTE that `[mod_<label>]` will have to stay in order to be backwards compatible :sob: